Description: Skip TRY_REQUIRE_PROG's runtime environment checks during packaging builds

Author: try packaging
Forwarded: not-needed
--- a/configure.ac
+++ b/configure.ac
@@ -82,18 +82,7 @@
 AC_PROG_INSTALL
 
 dnl TRY_REQUIRE_PROG(progname, checking_msg, pre, true_if_failed, post, fail_msg)
-AC_DEFUN([TRY_REQUIRE_PROG], [
-AC_MSG_CHECKING([ifelse([$2], [], [for $1], [$2])])
-$3
-if ifelse([$4], [], [! type -p $1 >/dev/null 2>&1], [$4])
-then
-        AC_MSG_RESULT([no])
-        AC_MSG_ERROR([try needs $1 to work, but ifelse([$6], [], [it could not be found], [$6]).])
-else
-        AC_MSG_RESULT([yes])
-        $5
-fi
-])
+AC_DEFUN([TRY_REQUIRE_PROG], [])
 
 # symlinks
 AC_PROG_LN_S
